lib/src/utils_isolate_stub.dart

```diff
@@ -2,5 +2,7 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:async';
```

Comment:
Rather than reintroduce this import, maybe just change the SDK requirement to 2.1.0?

Reply:
We already pushed `1.0.7` that supports `2.0.0` – and is broken. Needed to push a fix for those folks.

Now we could entertain dropping SDK `2.0.0` in v1.0.9 of this package